### PR TITLE
Add Anya to the student list

### PR DESCRIPTION
**Pull request by darkthirteen**
Add Anya to the student list

### CODE REVIEW

**Comment by darkthirteen** [comment]
Add Anya to the student list

**Comment by ava8katushka** [comment]
Please, resolve the conflicts and change the name of the pull request to be more specific, than "Update README.md"

**Comment by ava8katushka** [comment]
Please, resolve the conflicts.

**Comment by ava8katushka** [comment]
Still conflicts :( Please resolve.

File: README.md [hunk 1]
```diff
@@ -13,3 +13,4 @@
 | 4 | Катя  | Такси                                  | 
 | 5 | Максим| Бойцовский клуб                        |
 | 6 | Мария | Однажды в Америке                      |
+| 5 | Аня   | Начало                                 |
```
Comment (ava8katushka) [inline README.md]: not 5, 7

**Comment by ava8katushka** [comment]
I'm sorry but while you were waiting new conflicts arrived.

**Comment by ava8katushka** [comment]
Please address the logical error and resolve the conflicts.

**Pull request merged**